src/modules/footer/components/attributionInfo/AttributionInfo.js

```diff
@@ -0,0 +1,56 @@
+import { html, nothing } from 'lit-html'; 
+import { BaElement } from '../../../BaElement';
+import { $injector } from '../../../../injection';
+
+/**
+ * a class for displaying the attribution of the basemap
+ * @class
+ * @author bakir_en 
+ */
+export class AttributionInfo extends BaElement {
+
+	constructor() {
+		super();
+		const { TranslationService, GeoResourceService } = $injector.inject('TranslationService', 'GeoResourceService');
+		this._translationService = TranslationService;
+		this._georesourceService = GeoResourceService;
+		this._content = null;
```

Comment:
Avoid unnecessary private fields, which makes your code more resistant to unwanted side-effects